WordPress/wordcamp.org
# Enhance next gen event budget tool - Days and Hours

See #976

This PR updated the Days and Hours fields as requested in the ticket.

### Screencast

https://github.com/WordPress/wordcamp.org/assets/18050944/fe86e1ab-f86a-4fd5-ba26-09e9aeb19d2f

### How to test the changes in this Pull Request:

1. Create a new next-gen event site through WordCamps >> Add new
2. Go to Budgets.
3. Play around with the Days and Hours dropdown and see if it behaves the same as the screencast above demonstrates.
4. Save what you've changed, and see if everything's saved.
5. Og wordcamp should remain everything the same. ie, `Days` remain pure text label.
